.github/workflows/pre-compile_llvm.yml

```diff
@@ -50,14 +50,18 @@ jobs:
           make --version
           ${{ matrix.cc }}-${{ matrix.version }} --version
           ${{ matrix.cpp }}-${{ matrix.version }} --version
+
+      - name: Extract branch name
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
```

Comment:
Please fix this the same way as in #12.

Reply:
Done. Sorry, I should have thought about this without you asking for it...